teos/src/watcher.rs

```diff
@@ -283,7 +282,7 @@ impl Watcher {
                     // DISCUSS: We could either free the slots or keep it occupied as if this was misbehavior.
```

Comment:
Rephrase this comment. We are referring to this as refunding now.

Reply:
I think we can remove it since it's now a general rule of thumb in other parts of the code base.
We can write this note in `GateKeeper::delete_appointments` documentation instead.